README.md

```diff
@@ -764,7 +777,7 @@ If you're opening issues related to these, please mention the version that the i
 License
 -------
 
-Pa11y is licensed under the [Lesser General Public License (LGPL-3.0)][info-license].  
+Pa11y is licensed under the [Lesser General Public License (LGPL-3.0)][info-license].
```

Comment:
This change will break the formatting of the README. A double space at the end of a line is a `<br/>`. I think as this keeps happening we should just add in an actual `<br/>` 😄 could you do that?